kong/konnect_application_service_test.go

```diff
@@ -0,0 +1,203 @@
+package kong
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/samber/lo"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestKonnectApplicationService_Create(T *testing.T) {
+	RunWhenDBMode(T, "postgres")
+	RunWhenEnterprise(T, ">=3.6.0", RequiredFeatures{})
+
+	assert := assert.New(T)
+	require := require.New(T)
+
+	client, err := NewTestClient(nil, nil)
+	assert.NoError(err)
+	assert.NotNil(client)
```

Comment:
Just to prevent test going any further when this fails:

```suggestion
	require.NoError(err)
	require.NotNil(client)
```